System.Reflection.Emit.AssemblybuilderSave.sln

```diff
@@ -3,9 +3,9 @@ Microsoft Visual Studio Solution File, Format Version 12.00
 # Visual Studio 15
 VisualStudioVersion = 15.0.26124.0
 MinimumVisualStudioVersion = 15.0.26124.0
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Experiment", "src\Experiment\src\Experiment.csproj", "{B7977360-6671-4707-9A1C-1C29D5BE2674}"
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "System.Reflection.Emit.AssemblybuilderSave", "src\Experiment\src\System.Reflection.Emit.AssemblybuilderSave.csproj", "{B7977360-6671-4707-9A1C-1C29D5BE2674}"
```

Comment:
The `System.Reflection.Emit` namespace in the system libraries contains types that you are going to be reimplementing. It is going to be confusing to deal with these conflicts.

I would recommend using different namespace to avoid these conflicts. For example, call this assembly `RuntimeLab.Reflection.Emit`.

Reply:
The other alternative is to only have the `RuntimeLab.Reflection.Emit` as the namespace, and have the NuGet package be one of the names that I suggested.

Reply:
How about `System.Reflection.Emit.Experimental`?

Reply:
(for both the package name and the namespace to use for the implementation)